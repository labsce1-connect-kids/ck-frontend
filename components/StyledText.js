import React from 'react';
import { Text } from 'react-native';
import * as Font from 'expo-font';


export function FuturaText(props) {
  return (
    <Text {...props} style={[props.style, { fontFamily: 'futura-light' }]} />
  );
}
