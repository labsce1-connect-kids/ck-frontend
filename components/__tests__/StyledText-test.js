import React from 'react';
import renderer from 'react-test-renderer';

import { FuturaText } from '../StyledText';

it(`renders correctly`, () => {
  const tree = renderer.create(<FuturaText>Snapshot test!</FuturaText>).toJSON();

  expect(tree).toMatchSnapshot();
});
