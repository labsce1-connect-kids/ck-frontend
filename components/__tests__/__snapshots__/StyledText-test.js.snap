// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Array [
      undefined,
      Object {
        "fontFamily": "futura-light",
      },
    ]
  }
>
  Snapshot test!
</Text>
`;
